python/Ganga/test/GPI/Bugs/TestSavannah9638.py

```diff
@@ -12,13 +12,11 @@ def wipe_temp_dir():
 
 class TestSavannah9638(GangaUnitTest):
     def setUp(self):
-        super(TestSavannah9638, self).setUp()
+        extra_opts = [('Configuration', 'gangadir', '/tmp/ganga_topdir-$USER')]
+        super(TestSavannah9638, self).setUp(extra_opts=extra_opts)
 
         wipe_temp_dir()
 
-        from Ganga.Utility.Config import setConfigOption
-        setConfigOption('Configuration', 'gangadir', '/tmp/ganga_topdir-$USER')
```

Comment:
For this to work, you need to shift the `wipe_temp_dir` call as well. That's what's causing the test failures.